near/omni-types/src/lib.rs

```diff
@@ -503,3 +520,44 @@ pub struct BasicMetadata {
     pub symbol: String,
     pub decimals: u8,
 }
+
+#[near(serializers=[borsh, json])]
+#[derive(Debug, Clone)]
+pub struct FastTransferId(pub [u8; 32]);
+
+#[near(serializers=[borsh, json])]
+#[derive(Debug, Clone)]
+pub struct FastTransfer {
+    pub transfer_id: TransferId,
+    pub token_id: AccountId,
+    pub amount: U128,
+    pub fee: Fee,
+    pub recipient: OmniAddress,
+    pub msg: String,
+}
+
+impl FastTransfer {
+    #[allow(clippy::missing_panics_doc)]
+    pub fn id(&self) -> FastTransferId {
+        FastTransferId(utils::keccak256(&borsh::to_vec(self).unwrap()))
```

Comment:
is there a reason why we use keccak instead of sha256 here?

Reply:
Not really. Is there a reason to use the opposite?

Reply:
@kiseln up

Reply:
Replaced 1f58c628b6b626b098da4d92171116c574c4cbcf